app/frontend/src/cljs/teet/contract/contract_common.cljs

```diff
@@ -57,43 +57,57 @@
       (contract-model/contract-name contract)]]
     [contract-external-links contract]]])
 
+(defn contract-partners-names [contract-with-partners]
+  (let [partners (:company-contract/_contract contract-with-partners)
+        names (reduce
+                (fn [acc item]
+                  (conj acc
+                    (get-in item [:company-contract/company :company/name])))
+                []
+                partners)]
```

Comment:
would `(mapv #(get-in % [:company-contract/company :company/name]))` do the same thing?